lib/core/text/flat.nit

```diff
@@ -933,12 +933,13 @@ class FlatBuffer
 	do
 		var c = capacity
 		if cap <= c then return
-		while c <= cap do c = c * 2 + 2
+		if cap <= 16 then c = 16
+		while c <= cap do c = c * 2
 		# The COW flag can be set at false here, since
 		# it does a copy of the current `Buffer`
 		written = false
 		var bln = _bytelen
-		var a = new NativeString(c+1)
+		var a = new NativeString(c)
```

Comment:
Yup I remember this one, though this is not used anymore since a `to_cstring` is allocated to avoid modification of the cstring from nit, therefore it is of no use anymore